### PR TITLE
refactoring of jp.co.yahoo.yosegi.message.objects

### PR DESCRIPTION
**Pull request by yohto**
### What is the necessity of this update? What is updated?
refactoring including modification of if -else if cascade to dispatcher using map.

### How did you do the test? (Not required for updating documents)
Path the maven test.

### CODE REVIEW

**Comment by yohto** [comment]
Could you please check jp.co.yahoo.yosegi.message.objects.BooleanObj: line 112.
It seems errata in original code, then I modified.

File: src/main/java/jp/co/yahoo/yosegi/message/objects/BooleanObj.java [hunk 1]
```diff
@@ -109,16 +104,32 @@ public void setBytes( final byte[] data ) throws IOException {
...
-      final byte[] data ,
-      final int start ,
-      final int length ) throws IOException {
-    value = 
+      final byte[] data,
+      final int start,
+      final int length) throws IOException {
+
+    /*
+     * Original code is not use start, but it seems start as buffer start point,
```
Comment (koijima) [inline src/main/java/jp/co/yahoo/yosegi/message/objects/BooleanObj.java]: It is a bug. Please delete the original code as it is as the comment.

Comment (yohto) [reply src/main/java/jp/co/yahoo/yosegi/message/objects/BooleanObj.java]: remove the comment

**Comment by koijima** [comment]
LGTM

**Pull request merged**